tests/test_bundle_init_bundle.py

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import subprocess
+import tempfile
+import unittest
+
+import torch
+
+from monai.networks.nets import UNet
+
+
+class TestBundleInit(unittest.TestCase):
```

Comment:
You can add `@skip_if_windows` decorator to skip the tests.
Refer to: https://github.com/Project-MONAI/MONAI/blob/dev/tests/test_bundle_ckpt_export.py#L30

Thanks.

Reply:
I saw that was elsewhere but I didn't get what was not Windows compatible, I'll add it here anyway.